interpreter/interpreter.go

```diff
@@ -177,6 +178,31 @@ func (itrp *Interpreter) doStatement(ctx context.Context, stmt semantic.Statemen
 	return nil, nil
 }
 
+// If the option is "now", evaluate the function and store in the execution
+// dependencies.
+func (irtp *Interpreter) evaluateNowOption(ctx context.Context, name string, init values.Value) {
```

Comment:
Is there a case where this might become more generic, to handle other options that need to be evaluated at execution? Not really a thing here, just wondering, as I build my internal representation of this patch.

Reply:
I'm not sure ... now is the only option I've ever used.

Reply:
Heh, same.